presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaSplitManager.java

```diff
@@ -151,6 +140,21 @@ private static String readSchema(String dataSchemaLocation)
         }
     }
 
+    private static InputStream openSchemaLocation(String dataSchemaLocation)
+            throws IOException
+    {
+        if (isURI(dataSchemaLocation.trim().toLowerCase(ENGLISH))) {
+            try {
+                return new URL(dataSchemaLocation).openStream();
+            }
+            catch (MalformedURLException e) {
+                // try again before failing
```

Comment:
is this comment needed? I find it misleading. I was expecting to see a repeat of `new URL(dataSchemaLocation).openStream()`.